### PR TITLE
CF Release

### PR DESCRIPTION
**Pull request by CFPBot**
### Added
- **capital-framework:** [patch] Add atomic-component dependency to main package.json.

### Changed
- **cf-expandables:** [minor] Update atomic-component dependency to 1.3.2.
- **cf-tables:** [minor] Update atomic-component dependency to 1.3.2.

### Removed
- **all components:** [patch] All individual component lockfiles.

## Review

- @cfpb/front-end-team-admin

## I am a bot

After this PR is merged, I will instruct Travis to [automagically](https://github.com/cfpb/capital-framework/tree/master/scripts/npm/prepublish) perform the following steps:

1. Increment Capital Framework's [version](https://github.com/cfpb/capital-framework/blob/canary/package.json#L3) per our [guidelines](https://github.com/cfpb/capital-framework/issues/179).
2. Add a timestamped entry to the [changelog](https://github.com/cfpb/capital-framework/blob/canary/CHANGELOG.md) with the new version and its changes.
3. [Tag](https://github.com/cfpb/capital-framework/tags) the release and push it to GitHub.
4. Publish both [capital-framework](https://www.npmjs.com/package/capital-framework) and any individually updated [components](http://cfpb.github.io/capital-framework/components/) to npm.
5. Update both [canary](https://github.com/cfpb/capital-framework/tree/canary), our development branch, and [master](https://github.com/cfpb/capital-framework/tree/master), our release branch.

If I do something wrong, [blame a human](https://github.com/cfpb/hubot-capital-framework/issues/new).

kitten gif

### CODE REVIEW

File: src/cf-notification/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-notification",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (Scotchester) [inline src/cf-notification/package.json]: I think we should manually correct this back down to 1.0 before performing this release.

Comment (Scotchester) [reply src/cf-notification/package.json]: I.e., 1.0 before this release, resulting in 1.0.1 with this release.

Comment (contolini) [reply src/cf-notification/package.json]: I originally thought npm releases can't be unpublished (cf-notifications has been published at v1.1.0) but it looks like they can if they're [less than 24 hours old](http://blog.npmjs.org/post/141905368000/changes-to-npms-unpublish-policy). 👍 

So I'll unpublish 1.1.0 and change the above line to 1.0.0.

File: src/cf-notification/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-notification",
-  "version": "1.1.1",
+  "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply src/cf-notification/package.json]: Yet it's currently published as 0.0.1, is that expected? https://www.npmjs.com/package/cf-notification

**Comment by contolini** [comment]
Yeah I republished it at 0.0.1 just to ensure it exists in the registry
before the bot releases new versions.

File: src/cf-notification/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (contolini) [reply src/cf-notification/package.json]: To clarify my below comment: I'll publish it now at 1.0.0 and move this to 1.0.1.

**Comment by contolini** [comment]
Alrighty cf-notification has been published at [1.0.0](https://www.npmjs.com/package/cf-notification) and it's listed in this PR as 1.0.1.

**Pull request merged**